modules/references/server/policies/references.server.policy.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var acl = require('acl'),
+    path = require('path'),
+    errorService = require(path.resolve('./modules/core/server/services/error.server.service'));
+
+acl = new acl(new acl.memoryBackend());
+
+exports.invokeRolesPolicies = function () {
+  acl.allow([{
+    roles: ['user', 'admin'],
+    allows: [{
+      resources: '/api/references',
+      permissions: ['post']
+    }]
+  }]);
+};
+
+exports.isAllowed = function (req, res, next) {
```

Comment:
very similar code is repeated over and over again

Reply:
I suspect this could be massively simplified, but perhaps not the right time in this PR.

Reply:
Opened a new issue for this #802